### DISCUSSION TITLE
Simple way to pass a single prop from one page to another?

### DISCUSSION THREAD

**Discussion by slikvik55** [quasarframework/quasar#9378] [General - Components / Directives / etc]
I have one value that I need to pass in the simple app from the setup page to the main page. I don't want to use Vuex as this app is super simple. I've currently got it passing as a prop on the route but I really don't want it overly exposed in the URL to the user.

I tried setting up a simple store but I can't get it working with Quasar's structure.
I was hoping I could just have it as a prop in the layout page and then set it on page 1 and get it on page 2, but I can't get it working. Any ideas or solutions would be appreciated!

Thanks

**Comment by ontwikkelfabriek** [comment]
@slikvik55 

This will work in Quasar V1(Vue2) and Quasar V2 (Vue3):
https://quasar.dev/quasar-plugins/web-storage

**Reply by slikvik55** [reply]
Thanks @ontwikkelfabriek !
